### PR TITLE
fixes: LT-2575, LT-2253

### PR DESCRIPTION
**Pull request by jfaracik**
* LT-2575 - Notification is no longer sent to internal users when an application is submitted
* LT-2253 - Selecting other on select the role of the third party question should force user to give a description
* Small code cleanup

### CODE REVIEW

File: audit_trail/managers.py [hunk 1]
```diff
@@ -23,23 +23,22 @@ def create(self, *args, **kwargs):
         from cases.models import Case
 
         target = kwargs.get("target")
+        actor = kwargs.get("actor")
+        send_notification = kwargs.pop("send_notification") if "send_notification" in kwargs else True
+        ignore_case_status = kwargs.pop("ignore_case_status") if "ignore_case_status" in kwargs else False
```
Comment (andytaylorkainos) [inline audit_trail/managers.py:28]: you can do kwargs.pop("ignore_case_status", False)

File: applications/libraries/application_helpers.py [hunk 2]
```diff
@@ -68,4 +68,6 @@ def create_submitted_audit(request, application, old_status):
                 "old": CaseStatusEnum.get_text(old_status),
             }
         },
+        ignore_case_status=True,
```
Comment (BenAndrew-Kainos) [inline applications/libraries/application_helpers.py:71]: What is ignore case status for (surely we shouldn't ignore it)

File: audit_trail/managers.py (hunk 1 above)
Comment (jfaracik) [reply audit_trail/managers.py:28]: Thanks! The more you know 🌟

File: applications/libraries/application_helpers.py (hunk 2 above)
Comment (jfaracik) [reply applications/libraries/application_helpers.py:71]: AuditManager doesn't create audit entries for cases with a status of `draft`, so there's an optional parameter `ignore_case_status` to ignore this check.

**Pull request merged**